implementation/src/main/java/io/smallrye/jwt/auth/principal/KeyLocationResolver.java

```diff
@@ -181,13 +182,13 @@ PublicKey getContentJwk(String content, String kid) throws JoseException {
         return null;
     }
 
-    private void loadContents() throws IOException {
+    private AbstractMap.SimpleImmutableEntry<HttpsJwks, String> loadContents() throws IOException {
```

Comment:
Lets keep the original code here to keep things simpler

Reply:
I did a little different